core/src/main/java/io/ddf/content/ViewHandler.java

```diff
@@ -137,13 +137,8 @@ public DDF removeColumns(List<String> columnNames) throws DDFException {
     }
 
     DDF newddf = this.project(columns);
-    if(this.getDDF().isMutable()) {
-      this.getDDF().updateInplace(newddf);
-      return this.getDDF();
-    } else {
-      newddf.getMetaDataHandler().copyFactor(this.getDDF());
-      return newddf;
-    }
+    newddf.getMetaDataHandler().copyFactor(this.getDDF());
```

Comment:
Need to note this in the description, current behavior of remove_columns in PyClient is in place, this will break current behavior